extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -21,7 +17,7 @@ $color-rail-background: mix($color-page, $black, 90%);
 }
 
 @mixin wikia-rail-element() {
-    background-color: $color-rail-background;
+    background-color: $color-new-module-background;
```

Comment:
The point is it will not be used only in Rail Module. It will be used across the wiki pages that's why is defined in 'skins/shared/color'